docs/source/rpc.rst

```diff
@@ -100,7 +100,7 @@ applications can always explicitly move the input tensors to CPU on the caller
 and move it to the desired devices on the callee if necessary.
 
 .. warning::
-  TorchScript support in RPC is experimental and subject to change.
+  TorchScript support in RPC is beta and subject to change.
```

Comment:
@jspisak - Is this beta or prototype? This doc shows prototype:https://fb.quip.com/7fGEAhAY5Etz

Reply:
Yes, the TorchScript/RPC is a prototype

Reply:
Can we also add torch.sparse to this PR as well?  https://pytorch.org/docs/stable/sparse.html?highlight=experimental#